unicodetools/pom.xml

```diff
@@ -0,0 +1,130 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>unicodetools</artifactId>
+	<groupId>org.unicode.unicodetools</groupId>
+	<name>Unicode Tools</name>
+
+	<version>0.0.1</version>
+
+	<parent>
+		<groupId>org.unicode.unicodetools</groupId>
+		<artifactId>unicodetools-parent</artifactId>
+		<version>0.0.0</version>
+	</parent>
+
+
+	<url>https://unicode.org/cldr</url>
+
+	<properties>
+		<maven.compiler.source>1.8</maven.compiler.source>
+		<maven.compiler.target>1.8</maven.compiler.target>
+		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+		<!-- <mainClass>org.unicode.cldr.tool.Main</mainClass> -->
+		<junit-version>5.7.2</junit-version>
+		<!-- Need to use an updated surefire plugin here. -->
+		<maven-surefire-plugin-version>3.0.0-M5</maven-surefire-plugin-version>
+	</properties>
+
+	<scm>
+		<connection>scm:git:https://github.com/unicode-org/unicodetools.git</connection>
+	</scm>
+
+	<!-- <parent>
+		<groupId></groupId>
+		<artifactId></artifactId>
+		<version></version>
+	</parent>
+ -->
+	<dependencies>
+		<dependency>
+			<groupId>com.ibm.icu</groupId>
+			<artifactId>icu4j-for-cldr</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>com.ibm.icu</groupId>
+			<artifactId>utilities-for-cldr</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.unicode.cldr</groupId>
+			<artifactId>cldr-code</artifactId>
+		</dependency>
+		<!-- test -->
+		<dependency>
+			<groupId>org.junit.jupiter</groupId>
+			<artifactId>junit-jupiter</artifactId>
+			<scope>test</scope>
+			<version>${junit-version}</version>
+		</dependency>
+
+		<!-- <dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>com.google.guava</groupId>
+			<artifactId>guava</artifactId>
+		</dependency> -->
+
+		<!-- <dependency>
+			<groupId>org.apache.ant</groupId>
+			<artifactId>ant</artifactId>
+			<scope>compile</scope>
+		</dependency>
+ -->
+		<!-- <dependency>
+			<groupId>xml-apis</groupId>
+			<artifactId>xml-apis</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>xerces</groupId>
+			<artifactId>xercesImpl</artifactId>
+		</dependency> -->
+
+	</dependencies>
+
+	<build>
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<artifactId>maven-surefire-plugin</artifactId>
+					<version>${maven-surefire-plugin-version}</version>
+					<configuration>
+						<systemPropertyVariables>
```

Comment:
optional: IIUC, you could add a comment to indicate that this section is creating variables in the environment that will be picked up & used by code later on. 

Is that understanding correct? If so, for my own knowledge, how are these Java property variables getting picked up? That could be useful to document.

If that understanding is incorrect, then it seems inconsistent to make these properties all-uppercase when `icu.version` and `cldr.version` and other config identifiers are lowercase.

Reply:
0. Yes, I'll add a comment "what is this", good idea.
1. These are for tests, not 'all code'.
2. `icu.version` is a Maven 'property', `UVERSION` is a Java property ( command line: `java -DUVERSION=1.2.3`.  `UVERSION` is pre-existing here. )

Reply:
Ok, thanks for the info. I like this move you made from this sub-project pom.xml to the root pom.xml, since these properties seem globally applicable, and thus should be configured once at the root and inherited (and optionally overridden) when needed.

Reply:
There are more dependencies in UnicodeJsps that could be moved up to the parent, could consider under  #131